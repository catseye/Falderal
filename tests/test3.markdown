Falderal Test 3
---------------

A Falderal document which is ill-formed.

    -> Functionality "Cat" is implemented by
    -> shell command "python cat.py"

    -> Tests for functionality "Cat"

Cat cats.

    | meow

Oops, I didn't include an expectation.
