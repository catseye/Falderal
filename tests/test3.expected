FalderalSyntaxError: line 14: test text must be followed by expectation or test input
